packages/sitecore-jss-react/src/components/Placeholder.test.tsx

```diff
@@ -101,7 +101,9 @@ describe('<Placeholder />', () => {
         const phKey = 'page-content';
 
         const renderedComponent = mount(
-          <Placeholder name={phKey} rendering={component} componentFactory={componentFactory} />
+          <SitecoreContext componentFactory={componentFactory}>
```

Comment:
I think context is always there for PlaceholderComponent. Now we need to wrap it with SitecoreContext in order for it to find the desired element.

Reply:
The layout data context was not there in unit tests before - and it's still not needed for most test scenarios

Reply:
Since now we wrap the placeholder component with `SitecoreContext` it is necessary to wrap each test with `SitecoreContext` otherwise it cannot get componentFactory. A common function is also not possible or maybe not very valuable since Placeholder Component accepts so many props and each test case tests different props so a common function would be a overkill.